internal/api/helpers.go

```diff
@@ -0,0 +1,56 @@
+package api
+
+import (
+	"encoding/json"
+	"net/http"
+	"time"
+
+	"github.com/nikaydo/final/internal/date"
+	"github.com/nikaydo/final/internal/model"
+)
+
+type MsgId struct {
+	Id int64 `json:"id"`
+}
+
+type MsgError struct {
+	Error string `json:"error"`
+}
+
+func writeJSONResponse(w http.ResponseWriter, data any, status int) {
+	w.Header().Set("Content-Type", "application/json; charset=UTF-8")
+	w.WriteHeader(status)
+	jsonData, err := json.Marshal(data)
+	if err != nil {
+		http.Error(w, err.Error(), status)
```

Comment:
Так как мы сообщаем о проблеме маршалинга
```suggestion
		http.Error(w, err.Error(), 500)
```